openshift/templates/aries-mediator-service/aries-mediator-service-deploy.yaml

```diff
@@ -40,6 +64,30 @@ objects:
           role: ${ROLE}
           app: ${APP_NAME}${SUFFIX}
           env: ${TAG_NAME}
+          
+  - kind: Route
+    apiVersion: v1
+    metadata:
+      name: ${NAME}${SUFFIX}-admin
+      labels:
+        name: ${NAME}${SUFFIX}-admin
+        app: ${APP_NAME}${SUFFIX}
+        app-group: ${APP_GROUP}
+        role: ${ROLE}
+        env: ${TAG_NAME}
+      annotations:
+        haproxy.router.openshift.io/timeout: ${ROUTE_TIMEOUT}
+    spec:
+      host: ${APPLICATION_DOMAIN}
+      port:
+        targetPort: ${NAME}${SUFFIX}-admin
```

Comment:
The name of the route and port imply this is an admin route.  However the use of `APPLICATION_DOMAIN` for the host implies otherwise.  If this is and admin route you should use `APPLICATION_ADMIN_DOMAIN` to stay consistent with our other agent configurations for BC Reg and OrgBook.
Use https://github.com/bcgov/orgbook-configurations/blob/main/openshift/templates/agent/agent-deploy.yaml as the reference.

Reply:
Change completed